composer.json

```diff
@@ -17,7 +17,8 @@
 	],
 	"require": {
 		"php": ">=5.3.3",
-		"composer/installers": "~1.0"
+		"composer/installers": "~1.0",
+		"monolog/monolog": "~1.15"
```

Comment:
@camspiers's point is valid: if my code works with 1.`<15` it would be nice to allow for that package. The main thing is that I haven't tested.

I'll check out an older version and re-test.